app/src/main/java/org/literacyapp/synchronization/Main2Activity.java

```diff
@@ -79,7 +81,23 @@ public void cancelDisconnect() {
 
     @Override
     public void connect(WifiP2pConfig config) {
+        Log.i(P.Tag, "Connect(WifiP2pConfig config) called.");
+        Toast.makeText(this,"Connecting..." ,Toast.LENGTH_LONG).show();
```

Comment:
Remember to remove Toast messages before deploying the app to production. Consider logging it instead of using a Toast.